src/vulkan-renderer/application.cpp

```diff
@@ -266,9 +256,7 @@ VkResult Application::load_octree_geometry() {
     world::Cube cube = world::Cube::parse(test);
     cube.make_reactive();
 
-    cube.on_change.connect([](world::Cube* c) {
-        spdlog::debug("THE WORLD (octree) HAS CHANGED!");
-    });
+    cube.on_change.connect([](world::Cube *c) { spdlog::debug("THE WORLD (octree) HAS CHANGED!"); });
```

Comment:
Note for our TODO: List, this should be a smart pointer.

Reply:
ok.